subjects/gatecrashers/README.md

```diff
@@ -0,0 +1,64 @@
+## Gatecrashers
+
+### Instructions
+
+Unfortunately many of your guests started to invite plus ones, that started to invite plus ones, that started... to be short, everybody is inviting everybody and the situation is rapidly going out of control.
+
+To fix this issue you will introduce a [Basic Access Authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) on your server.
+
+To modify the guest list now your friends will need to make authenticated requests. You decided that only your 3 best friends (`Caleb_Squires`, `Tyrique_Dalton` and `Rahima_Young`) will be able to modify the list. You also told them the secret password `abracadabra` in order to do that.
+
+The request query will look like this `curl -u Caleb_Squires:abracadabra ...`.
+
+The server should properly handle unauthorized requests using the error code `401`.
+
+Apart for the authentication part your server's behavior should remain unchanged:
+- You will listen to port `5000` and print a message containing the port you are using.
+- For each **authorized** http `POST` request, your program should create the corresponding JSON file and store the contents of the body, and then provide the content as JSON in the HTTP response, if possible. 
+
+### Example
+
+To test your program, you should be able to expect the following behavior once your program is up and running.
+
+Unauthorized attempt:
+
+```shell
+curl -i -X POST localhost:5000/Ricky_Banni -H "Content-Type: application/json" -d '{"answer": "yes", "drink": "alcohol", "food": "bats"}'
+HTTP/1.1 401 Unauthorized
+Content-Type: text/html
```

Comment:
this should be `application/json` to be consistent with the tests

Reply:
done